mongodb/activity/queryDocument/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/project-flogo/datastore-contrib/mongodb/activity/queryDocument
```

Comment:
And is this the structure you decided? `mongodb/activity` and not `activity/mongodb`?

Reply:
There has been a conversation between FE team (Vijay, Tracy) and OSS team (Matt, Frank) regarding the structure for such contributions and this is the structure that has been decided upon.